index.js

```diff
@@ -490,14 +490,31 @@ exports.init = function (sbot, config) {
 
     function next(accept) {
       getConfirm(invite_id, accept, function (err, confirm) {
-        if(!confirm)
-          connectFirst(invite, function (err, rpc) {
-            if(err) return cb(err)
-            rpc.peerInvites.confirm(accept, function (err, confirm) {
-              //TODO: store confirms for us in the state.
-              cb(err, confirm)
+        if (err) return cb(err)
+        if(confirm) {
+          console.error('ssb-peer-invites: Invite already confirmed')
+          return // mix: send something else back in cb for client side?
+        }
+
+        connectFirst(invite, function (err, rpc) {
+          if(err) return cb(err)
+          rpc.peerInvites.confirm(accept, function (err, confirm) {
+            //TODO: store confirms for us in the state.
+            if (err) return cb(err)
+
+            const addr = invite.pubs
+              .filter(Boolean)
+              .find(pubAddress => pubAddress.indexOf(accept.author) > -1)
+            //mix: this logic should be extracted
```

Comment:
This piece in particular needs close review. I don't know if this is how to get the right sort of address for `sbot.gossip.connect`

It was a little tricky knowing what the form of all the things being passed around was - `invite`, `accept` etc